.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### All submissions
+
+* [ ] I have followed the [Contributing Guidelines](https://github.com/lra/mackup/blob/master/.github/CONTRIBUTING.md)
+* [ ] I have checked to ensure there aren't other open [Pull Requests](https://github.com/lra/mackup/pulls) for the same update/change
+
+### Adding/updating Application X Support
+
+* [ ] This PR is only for one application
+* [ ] It has been added to the list of supported applications in the [README](https://github.com/lra/mackup/blob/master/README.md)
+* [ ] Changes have been added to the WIP section of the [CHANGELOG](https://github.com/lra/mackup/blob/master/CHANGELOG.md)
+
+- [ ] Syncing does not break the application
```

Comment:
why the newline and the switch between `*` and `-`?

Reply:
The newline was just to mark a difference between GitHub-specific requirements vs config file requirements, and the change from `*` to `-` allowed for the line break.

However, I'll remove it if that's preferred :)